packages/react-native-navigation/src/__tests__/rum/instrumentation/DdRumReactNativeNavigationTracking.test.tsx

```diff
@@ -120,6 +121,28 @@ it('M send a RUM ViewEvent W startTracking() componentDidAppear', async () => {
 
 
 
+it('M send a RUM ViewEvent W startTracking() componentDidAppear { custom viewPredicate }', async () => {
+    // GIVEN
+    let componentId = "component42"
+    const customViewName = "custom_view_name"
+    const predicate: ViewNamePredicate = function (_event: ComponentDidAppearEvent, _trackedName: string)  { 
```

Comment:
minor, can be even shorter if both arguments not used:

```
const predicate: ViewNamePredicate = () => {
   return customViewName
}
```

but no need to change

Reply:
I'd rather keep things with explicit type. That's probably trust issue against javascript, but I prefer to keep things explicit.